content/posts/2023/previewing-workspace-files.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Previewing workspace files"
+date: 2023-11-13T14:00:00+01:00
+---
+
+For a long time, the REANA web UI has allowed users to preview certain types of files present in the
+workflow workspace. This has proven to be very convenient for users who want to quickly glance through
+images and text files in their workspaces without having to download them. 
+Expanding on this, recent enhancements in REANA now include the previewing of PDF and ROOT files, 
+increasing the number of file types that can be inspected within the browser.
+
+<!--more-->
+
+## Images and text files preview
+
+Probably amongst the most common file types in a workflow workspace are images and text files. The web UI has
+always allowed to preview them from a modal window opened just by clicking on the name of the file in the workspace list.
+From there, you can also download the file if needed. Note that text files are not limited to `.txt` files, 
+but they also include logs, code, and other text-based files.
+
+![Text file preview](/images/text-file-preview.png)
+
+## HTML files preview
+
+Since REANA 0.8.1, users have been able to preview HTML files directly in the web UI.
+This feature remains a valuable asset, allowing for example immediate visibility into Snakemake
+reports and other HTML-based documentation pages directly within the browser.
+
+![HTML file preview](/images/snakemake-support-report.png)
+
+## PDF files preview
+
+Starting from REANA 0.9.1, when you click on a PDF file in your workflow's workspace in the web UI, an embedded
+PDF viewer will load the document. You will be able to scroll through pages, zoom in and out,
+or even download the PDF if necessary.
+You can use it to quickly inspect the content of any PDF generated by your analysis or uploaded to your
+workspace.
+Here is a sneak peek of how it looks:
+
+![PDF file preview](/images/pdf-files-preview.png)
+
+## ROOT files preview
+
+[ROOT](https://root.cern.ch) files can also be previewed directly in the web UI.
+We have embedded a ROOT file viewer, based on [JSROOT](https://root.cern/js/), enabling you to inspect your
+ROOT files without leaving your browser.
+You can navigate through the file structure, interact with charts and ROOT objects, zoom on some details,
+and download the file if needed.
+Here you can see the ROOT file viewer in action:
+
+![ROOT file preview](/images/reana-0.9.1-root-preview.png)
+
+These enhancements aim to save time and streamline the process of navigating and inspecting files
```

Comment:
In the optics of moving from news-announcement style to teaching-blog-post style, you could leave out the phrase "These announcements..." and directly end the blog post with "We are continually improving...". This could go simply like this:

- We hope that the interactive file preview feature is proving to be beneficial in your usage of the REANA web interface. We are striving to continually improve the usability of the system based on your feedback and suggestions. Pease let us know if you would like to support previewing more file types!